packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -288,11 +253,24 @@ const ModalScrollContent = styled(Flex)<ModalScrollContentProps>`
   `};
 `
 
-const CloseIconWrapper = styled(Box)`
+// TODO: Consider extracting the equivalent of `TouchableWithoutFeedback`
+// TODO: Icon should support `currentColor` as a valid color
+const CloseButton = styled.button`
   position: fixed;
-  top: ${space(2)}px;
-  right: ${space(2)}px;
+  appearance: none;
+  top: 0;
+  right: 0;
   cursor: pointer;
+  padding: ${space(2)}px ${space(2)}px 0 0;
+  background-color: transparent;
+  border: 0;
+  color: ${color("black100")};
+
+  &:focus {
+    path {
+      fill: ${color("black60")};
```

Comment:
In general I think it's a bit simpler for Icons to just be `currentColor` by default which makes them simpler to control from the outside.

Reply:
We should probably eventually update that.